packages/ketcher-core/src/application/editor/actions/template.ts

```diff
@@ -308,22 +323,45 @@ function fromTemplateOnBond(restruct, template, bid, flip) {
       const newBondId = operation.data.bid
 
       if (isFusingBenzeneBySpecialRules) {
-        const newBondType = benzeneDoubleBondIndexes.includes(tBondIndex)
-          ? Bond.PATTERN.TYPE.DOUBLE
-          : Bond.PATTERN.TYPE.SINGLE
-        action.addOp(
-          new BondAttr(newBondId, 'type', newBondType).perform(restruct)
-        )
+        const isBenzeneTemplate = tmpl.name === benzeneMoleculeName
+        const isCyclopentadieneTemplate =
+          tmpl.name === cyclopentadieneMoleculeName
+        if (isBenzeneTemplate) {
+          const newBondType = benzeneDoubleBondIndexes.includes(tBondIndex)
+            ? Bond.PATTERN.TYPE.DOUBLE
+            : Bond.PATTERN.TYPE.SINGLE
+          action.addOp(
+            new BondAttr(newBondId, 'type', newBondType).perform(restruct)
+          )
+        }
+        if (isCyclopentadieneTemplate) {
+          const { beginBondIds, endBondIds } = Bond.getBondNeighbourIds(
+            struct,
+            bid
+          )
+          const bondBegin = struct.bonds.get(beginBondIds[0])!
```

Comment:
We have an eslint rule which disables using non null assertion. Could we fix the eslint error? 
<img width="767" alt="image" src="https://github.com/epam/ketcher/assets/100932091/5bd005ba-048d-429f-9afb-f11d940696e4">